### PR TITLE
[WIP] Adds offboarding checklist

### PR DESCRIPTION
**Pull request by mogul**
Pending checks from Liberator and Skyporter perspective

### CODE REVIEW

**Comment by suprenant** [comment]
Just commenting here to say that it's nice to see this – @juliaelman and I were just wondering about offboarding, and I was going to ask you about it @mogul

**Comment by mogul** [comment]
@suprenant: We need to do it for compliance purposes in any case. ;) 

- [ ] @msecret: Please add mention of any special access granted via Liberator squad activity that should be revoked, then check here
- [ ] @dlapiduz @nikzei @berndverst: Please add mention of any special access granted via SkyPorter squad activity that should be revoked, then check here

Once both boxes above are checked, this can be merged.

**Comment by msecret** [comment]
This covers everything from liberator

**Comment by dlapiduz** [comment]
@mogul can you add Trello?

**Comment by berndverst** [comment]
When should this trigger? When an employee leaves the cloud.gov project / team, or when they leave 18F? Probably the former. If so, should the process be initiated by the person's PM?

And not everybody on the larger cloud.gov team would have permission to complete all steps.

File: OffboardingChecklist.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+6. Submit the issue
+7. Assign the issue to the person who bravely volunteered to be handle the person's offboarding
+8. Put the issue into the "In Progress" pipeline in ZenHub
+
+---
+
+In order to complete LeavingPerson's exit from the cloud.gov team, the assignee should complete a prescribed set of tasks that will remove any special access.
+
+# Directions:
+**Assignee:** If you can’t complete any of the items on your checklist personally, _you are responsible for ensuring that an appropriate person does it_.
```
Comment (mogul) [inline OffboardingChecklist.md]: @berndverst: We're explicit about what to do in that case here.

**Pull request merged**